src/components/Login.jsx

```diff
@@ -1,56 +1,77 @@
 import React from "react";
-import ReactDOM from "react-dom";
 
-function Login(){
+import "./Login.css";
 
-  return(
+const Login = () => {
+  console.log("Login called");
+  return (
     <div>
-  <div className = "abc">
```

Comment:
fixing indentation, I use a vscode extension for that, https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode so you can use a shortcut to auto-format the whole file (option + shift + f on OSX)